telegramium-high/src/main/scala/telegramium/bots/high/WebhookBot.scala

```diff
@@ -130,31 +131,52 @@ abstract class WebhookBot[F[_]: ConcurrentEffect: ContextShift](
   ): F[Unit] =
     bot.execute(setWebhook(url, certificate, ipAddress, maxConnections, allowedUpdates)).void
 
-  private def createServer(executionContext: ExecutionContext): Resource[F, Server[F]] = {
+  private def routes(): HttpRoutes[F] = {
     val dsl = Http4sDsl[F]
     import dsl._
 
-    def app(): HttpApp[F] =
-      HttpRoutes
-        .of[F] { case req @ POST -> BotPath =>
-          handleUpdateReq(req).flatMap {
-            _.fold(Ok()) { m =>
-              val inputPartFiles = m.payload.files.collect { case (filename, InputPartFile(file)) =>
-                (filename, file)
-              }
-              val attachments = toFileDataParts(inputPartFiles, blocker)
-              if (attachments.isEmpty)
-                Ok(m.payload.json)
-              else {
-                val parts = toMultipartWithFormData(m.payload.json, inputPartFiles.keys.toList, attachments)
-                Ok(parts).map(_.withHeaders(parts.headers))
-              }
-            }
+    HttpRoutes.of[F] { case req @ POST -> BotPath =>
+      handleUpdateReq(req).flatMap {
+        _.fold(Ok()) { m =>
+          val inputPartFiles = m.payload.files.collect { case (filename, InputPartFile(file)) =>
+            (filename, file)
+          }
+          val attachments = toFileDataParts(inputPartFiles, blocker)
+          if (attachments.isEmpty)
+            Ok(m.payload.json)
+          else {
+            val parts = toMultipartWithFormData(m.payload.json, inputPartFiles.keys.toList, attachments)
+            Ok(parts).map(_.withHeaders(parts.headers))
           }
         }
-        .orNotFound
+      }
+    }
+  }
+
+  private def createServer(executionContext: ExecutionContext): Resource[F, Server[F]] =
+    BlazeServerBuilder[F](executionContext).bindHttp(port, host).withHttpApp(routes().orNotFound).resource
+
+}
+
+object WebhookBot {
+
+  def compose[F[_]: ConcurrentEffect: Timer](
```

Comment:
Would be nice to add some scaladocs and mention the method in the README.

Reply:
Done